src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx

```diff
@@ -9,11 +9,11 @@ import infrastructureAzureMonitorMetricsIntegration from 'images/azure-monitor-m
 import infrastructureAzureMonitorMetricsConfigure from 'images/azure-monitor-configure.webp'
 import infrastructureAzureMonitorEnableToggle from 'images/azure-monitor-enable-toggle.webp'
 
-Monitor and report data about your Microsoft Azure services to New Relic with the New Relic Azure Monitor integration.
+Monitor and report data about your Microsoft Azure services to New Relic with the New Relic Azure monitor integration.
```

Comment:
Is 'monitor' the right casing? MSFT uses 'Monitor'(https://learn.microsoft.com/en-us/azure/azure-monitor/overview, https://docs.newrelic.com/docs/release-notes/infrastructure-release-notes/cloud-integration-release-notes/) and New Relic also uses 'Monitor' (Infrastructure -> Azure. The integration is called 'Azure Monitor metrics').
NR release notes also uses 'Monitor': https://docs.newrelic.com/docs/release-notes/infrastructure-release-notes/cloud-integration-release-notes/